src/components/Modal/index.js

```diff
@@ -49,6 +49,11 @@ export default class Modal extends PureComponent {
       onClose,
     } = this.props
 
+    if (source === 'backdrop') {
```

Comment:
is there a reason this should only happen with the backdrop click?

Reply:
Actually I added that because I wasn't 100% sure if that would have impact on other use cases...
Would it be safe to remove that `if`?

Reply:
let's go ahead and remove it, and if we find any issues later we can fix them.  but i don't see why there would be any and prefer to not protect ourselves from something we aren't even sure exists